addons/account/models/res_config_settings.py

```diff
@@ -184,6 +189,11 @@ def _inverse_account_default_credit_limit(self):
                 setting.account_default_credit_limit,
                 self.company_id.id
             )
+    @api.depends('company_id')
+    def _compute_terms_url(self):
+        for record in self:
+            if record.module_website:
+                record.terms_url = url_join(record.website_id.get_base_url(), '/terms')
```

Comment:
If we know what the `record` is, we should state it so it's obvious what we are working with.

For example, the ORM team works with object that could be anything from `account.move.line` to `res.settings` and thus, they need to put `record` but we don't.

The goal is for us to make the code as understandable as possible. (without needing to look around the code if possible)
```suggestion
    def _compute_terms_url(self):
        for setting in self:
            if setting.module_website:
                setting.terms_url = url_join(setting.website_id.get_base_url(), '/terms')
```